### DISCUSSION TITLE
Support for mbelib

### DISCUSSION THREAD

**Discussion by fventuri** [jketterl/codecserver#5] [General]

**Comment by jketterl** [comment]
Well, as you know, I cannot support mbelib any more, no matter in what form. I'm not even sure if I can actually leave this thread on here.

What I can tell you is that you shouldn't need to fork the whole project to be able to add a module.

**Comment by fventuri** [comment]
Jakob,
I don't think I can delete this discussion, but I removed all the content from my post.

I'll work on just a module for those who are interested then.

Thanks,
Franco
